portfolio/src/main/webapp/script.js

```diff
@@ -207,7 +211,7 @@ function addComment() {
  * Delete all comment data and clear main page.
  */
 function deleteAllComments() {
-  if (window.confirm("Careful! Do you really want to delete all comments?")) { 
+  if (window.confirm("Careful! Do you really want to delete all comments?")) {
```

Comment:
Single quotes in place of double.
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

[Optional]:
Rather than inlining this comment, consider defining constants like this at the top of your file, or in another file.

```javascript
const messages = {
  deleteAllCommentsWarning: '...'
}
```

or 

```javascript
const DELETE_ALL_COMMENTS_WARNING = '...';
```

Reply:
This recommendation would not include your arrays of strings, which can continue to remain in the functions, or can be moved to the top of the file independent of this recommendation to avoid instantiating the array every time the function is called.